src/lib/hooks.js

```diff
@@ -69,6 +69,19 @@ const api = {
 		updateIframeGuestData({ department });
 	},
 
+	async setBusinessUnit(newBusinessUnit) {
+		if (!newBusinessUnit || !newBusinessUnit.trim().length) {
+			throw new Error('Error! Invalid business ids');
+		}
+
+		const { businessUnit: existingBusinessUnit } = store.state;
+		if (existingBusinessUnit === newBusinessUnit) {
+			return;
+		}
+		store.setState({ businessUnit: newBusinessUnit });
+		await loadConfig();
+	},
+
 	clearDepartment() {
```

Comment:
The ability to clear Business Units is missing, a method for that is needed.

Reply:
Good point! added `clearBusinessUnit` method for that